monkestation/code/modules/ranching/chickens/_chicken.dm

```diff
@@ -0,0 +1,458 @@
+/mob/living/simple_animal/chick
+	name = "\improper chick"
+	desc = "Adorable! They make such a racket though."
+	icon = 'monkestation/icons/mob/ranching/chickens.dmi'
+	icon_state = "chick"
+	icon_living = "chick"
+	icon_dead = "dead_state"
+	icon_gib = "chick_gib"
+	can_be_held = TRUE
+	worn_slot_flags = ITEM_SLOT_HEAD
+	held_state = "chick"
```

Comment:
Probably better to not have any chickens carried since so many have active effects